scout/adapter/mongo/variant.py

```diff
@@ -702,12 +708,22 @@ def get_variants_hgnc_overlapping(
         if not limit:
             limit = 30 if variant_obj["category"] == "snv" else 45
 
+        variant_carriers = self.get_variant_carriers(variant_obj)
+
         query = {
             "$and": [
                 {"case_id": variant_obj["case_id"]},
                 {"category": category},
                 {"variant_type": variant_type},
                 {"hgnc_ids": {"$in": hgnc_ids}},
+                {
```

Comment:
Liking this better right now: https://github.com/Clinical-Genomics/scout/blob/0bfb415f6de01a8bea2f355e63830f03aade1348/scout/adapter/mongo/variant.py#L494

Reply:
Yes, thank you - this makes sense to me given that we are to impose limits here. I guess gene list matching might have been nice as well, but it should be ok to leave that. They carry the variant of origin anyway.